src/views/BusinessCase/index.tsx

```diff
@@ -70,9 +79,69 @@ export const BusinessCase = () => {
     }
   ]);
 
+  const businessCase = useSelector(
+    (state: AppState) => state.businessCase.form
+  );
+
+  const isLoading = useSelector(
+    (state: AppState) => state.businessCase.isLoading
+  );
+
+  const isSaving = useSelector(
+    (state: AppState) => state.businessCase.isSaving
+  );
+
   const [pageIndex, setPageIndex] = useState(0);
   const pageObj = pages[pageIndex];
 
+  const dispatchSave = () => {
+    const { current }: { current: FormikProps<BusinessCaseModel> } = formikRef;
+    if (current && current.dirty && !isSaving) {
+      if (businessCaseId === 'new') {
+        current.resetForm({ values: current.values, errors: current.errors });
```

Comment:
This isn't having the intended effect of forcing the form to save if it returns to its initial state. I'm investigating.

Reply:
I made a 1-line change that fixes this without (I think) breaking anything else.